docs/docs/authenticate/saml/google.mdx

```diff
@@ -42,15 +42,16 @@ to copy-paste the contents of this document into the Firezone portal in the next
-| Metadata                  | see note       | Paste the contents of the SAML metadata document you downloaded in the previous step from Google.                                      |
-| Sign assertions           | Checked.       |                                                                                                                                        |
-| Sign metadata             | Checked.       |                                                                                                                                        |
-| Require signed assertions | Checked.       |                                                                                                                                        |
-| Required signed envelopes | **Unchecked.** |                                                                                                                                        |
+| Setting                     | Value            | Notes                                                                                                                                       |
+| --------------------------- | ---------------- | ----------------------------------------------------------------------------------------------------------------------------------------    |
+| Config ID                   | google           | Firezone uses this value to construct endpoints required in the SAML authentication flow (e.g., receiving assertions, login requests).      |
+| Label                       | Google           | Appears on the sign in button for authentication.                                                                                           |
+| Metadata                    | see note         | Paste the contents of the SAML metadata document you downloaded in the previous step from Google.                                           |
+| Sign assertions             | Checked.         |                                                                                                                                             |
+| Sign metadata               | Checked.         |                                                                                                                                             |
+| Require signed assertions   | Checked.         |                                                                                                                                             |
+| Required signed envelopes   | **Unchecked.**   |                                                                                                                                             |
+| Auto create users           | Default `false`  | Enable this setting to automatically create users when signing in with this connector for the first time. Disable to manually create users. |
```

Comment:
This field doesn't have a default value, it's always for explicit user selection.
```suggestion
| Auto create users           |   | Enable this setting to automatically create users when signing in with this connector for the first time. Disable to manually create users. |
```

Reply:
Ah, we have a checkbox on UI so effectively it's false by default, 🤔.